### PR TITLE
updated the flux config changes

### PR DESCRIPTION
**Pull request by hmcts-kodatala**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: apps/fis/fis-hmc-api/ithc.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      replicas: 2
+      memoryRequests: "768Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "3072Mi"
+      cpuLimits: "2500m"
+      readinessDelay: 45
+      readinessTimeout: 15
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/fis/hmc-api:prod-f2e9743-20230111182430 #{"$imagepolicy": "flux-system:prl-cos"}
```
Comment (AlokDatta) [inline apps/fis/fis-hmc-api/ithc.yaml:18]: Image policy needs to be your image policy

File: apps/fis/fis-hmc-api/ithc.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      environment:
+        FEATURE_EXAMPLE: true
+
+    global:
+      environment: ithc
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
```
Comment (AlokDatta) [inline apps/fis/fis-hmc-api/ithc.yaml:24]: Verify the tennant id as well. It is available in fis-aat MI

File: apps/fis/fis-hmc-api/ithc.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      enableKeyVaults: true
```
Comment (AlokDatta) [inline apps/fis/fis-hmc-api/ithc.yaml:25]: You can overrirde secrets if you want with this. We have done this for citizen

File: apps/fis/fis-hmc-api/perftest.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      replicas: 2
+      memoryRequests: "768Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "3072Mi"
+      cpuLimits: "2500m"
+      readinessDelay: 45
+      readinessTimeout: 15
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/fis/hmc-api:prod-f2e9743-20230111182430 #{"$imagepolicy": "flux-system:prl-cos"}
```
Comment (AlokDatta) [inline apps/fis/fis-hmc-api/perftest.yaml:19]: Image policy. You need to create own image policy. Not prl-cos

File: apps/fis/fis-hmc-api/prod.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      replicas: 2
+      memoryRequests: "768Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "3072Mi"
+      cpuLimits: "2500m"
+      readinessDelay: 45
+      readinessTimeout: 10
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/fis/hmc-api:prod-f2e9743-20230111182430 #{"$imagepolicy": "flux-system:prl-cos"}
```
Comment (AlokDatta) [inline apps/fis/fis-hmc-api/prod.yaml:19]: Same as above

File: apps/fis/perftest/base/kustomization.yaml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../../rbac/nonprod-role.yaml
+namespace: fis
```
Comment (AlokDatta) [inline apps/fis/perftest/base/kustomization.yaml:6]: Make changes to automation folder too to automate the process. Else it won't reflect

**Review by msl8r — APPROVED** [review]

**Review by jordankainos — DISMISSED** [review]
lgtm

**Review by msl8r — DISMISSED** [review]

**Review by yogendraU — APPROVED** [review]

**Review by msl8r — APPROVED** [review]

**Pull request merged**